### ISSUE TITLE
Use rss <author> in rss feeds

### ISSUE THREAD

**Issue by lionirdeadman** [mastodon/mastodon#18925]
### Pitch

The RSS feeds are sometimes made for individuals where the spec does say it can be omitted, however it is also not included in the tags RSS feeds. I think it would be good to use it for both to give a fuller picture of what the feed is representing.

### Motivation

Some feed readers show the author of a rss item, this can help me get a better idea of who made the post/article without going to the source manually in a tag RSS feed where multiple authors are expected.

Further, some feed readers allow for filtering based on the author information and this can be incredibly helpful in tag RSS feeds where I want to subscribe to a particular subject but where I do not like certain authors posting on it.

**Comment by vordenken** [comment]
I myself like the "old" way mastodon done RSS feeds way better. Now its just time and date? But why? Every rss-reader I know already shows the time and date of the post anyways? Name and or Source should be shown there...

**Comment by psyciknz** [comment]
Yes, was going to open same issue.   Especially helpful if an RSS feed is from a tag where there are multiple users

**Comment by Netpilgrim** [comment]
Just noticed the missing author info in an RSS feed for #threatintel where I would like to be able to filter out authors like @zuckerberg@hackers.town.
